Dockerfile

```diff
@@ -127,6 +127,10 @@ RUN /root/.bitrise/tools/stepman update
 
 COPY ./ssh/config /root/.ssh/config
 
+# ------------------------------------------------------
+# --- Install Rubygems
+
+RUN gem update --system --no-document
```

Comment:
Actually you're right, just tested it here, with the latest VMs (were deployed a couple of hours ago): https://github.com/bitrise-tools/datapi-client/pull/2